lib/views/goto_page.dart

```diff
@@ -0,0 +1,193 @@
+                              color: Colors.white,
+                              onPressed: () {
+                                Navigator.pop(context);
+                              }),
+                        ),
+                      ),
+                      Expanded(
+                        flex: 1,
+                        child: IconButton(
+                            icon: Icon(Icons.directions_transit),
+                            color: Colors.black,
+                            onPressed: () {
+                              Navigator.pop(context);
+                            }),
+                      ),
+                      Expanded(
+                        flex: 1,
+                        child: IconButton(
+                            icon: Icon(Icons.directions_bike),
+                            color: Colors.black,
+                            onPressed: () {
+                              Navigator.pop(context);
+                            }),
+                      ),
+                      Expanded(
+                        flex: 1,
+                        child: IconButton(
+                            icon: Icon(Icons.directions_walk),
+                            color: Colors.black,
+                            onPressed: () {
+                              Navigator.pop(context);
+                            }),
+                      ),
+                      Expanded(
+                        flex: 1,
+                        child: Container(),
+                      ),
+                    ],
+                  ),
+                  Divider(),
+                ],
+              ),
+            ),
+          ],
+        ),
+        floatingActionButton: FloatingActionButton(
+            backgroundColor: constant.COLOR_CONCORDIA,
+            foregroundColor: Colors.white,
+            child: Text("GO"),
+            onPressed: () {
+              _searchedDestination == null
+                  ? _showToast(context)
+                  : {
+                      _searchedStart == null
+                          ? widget.confirmDirection(new List<Coordinate>.from([
+                              //this is to return the current location however there is a type conflict
+                              //unable to test further because current location needs outside navigation
+                              //TODO: update to the right type
+                              //widget._current,
+                              _searchedDestination
+                            ])) //this doesn't work because of outside isnt implemented yet
+                          : widget.confirmDirection(new List<Coordinate>.from(
+                              [_searchedStart, _searchedDestination])),
+                      Navigator.pop(context)
+                    };
+            }),
+      ),
+    );
+  }
+
+  //The snackbar is current not working but is a QoL implementation, will be added as a bug
```

Comment:
It might be a better idea to just remove this code completely if it's just an added feature.

Reply:
Gotcha, it is removed on the latest commit